### ISSUE TITLE
Is there way to login in kiali using token?

### ISSUE THREAD

**Issue by harshul4274** [kiali/kiali#2237]
Hello All,
I am using kiali to check the application status but i want to add a different login user to see the status. But i can not add a login part as a token login. I can't login as a i pass 

![kiali](https://user-images.githubusercontent.com/34963521/74311557-c2176380-4d95-11ea-8af6-63686856ff27.png)

**Versions used**
Kiali: v1.4
Istio: 1.4.0
Kubernetes flavour and version: v1.14.2

**To Reproduce**
1. Go to 'my browser'
2. browse a Kiali through my URL
3. As a you show a image i can't login
4. See below configmap for auth login strategy is token.

**Expected behavior**

```yaml
 apiVersion: v1
 data:
   config.yaml: |
     istio_namespace: istio-system
     auth:
       strategy: token
     server:
       port: 20001
       web_root: /kiali
     external_services:
       tracing:
         url: 
       grafana:
         url: http://grafana.istio-system:3000
         enabled: true
         auth:
           type: "basic"
           username: "grafana_username"
           password: "grafana_password"
           use_kiali_token: false
           token: ""
           ca_file: ""
           insecure_skip_verify: true
       prometheus:
         url: http://svc-infra-prometheus:9090
 kind: ConfigMap
 metadata:
   labels:
     app: kiali
     chart: kiali
     heritage: Tiller
     release: istio
   name: kiali
   namespace: istio-system
```

**Comment by jmazzitelli** [comment]
Kiali v1.4 does not support "token" auth strategy . You must upgrade to obtain that feature.

The "token" auth strategy was added via this PR: https://github.com/kiali/kiali/pull/1996 (commit https://github.com/kiali/kiali/commit/48921eb5146cb35eabc936a4048c83679330f8f6)

Looks like this was released in Kiali v1.11, so that is the minimum version you need to be able to use the token auth strategy.

**Comment by harshul4274** [comment]
Thanks for help me @jmazzitelli. Now, i solved my issue.
